tests/python/rest_api/test_tasks.py

```diff
@@ -820,6 +820,53 @@ def test_can_create_task_with_exif_rotated_images(self):
                         # original is 480x640 with 90/-90 degrees rotation
                         assert im.height == 640 and im.width == 480
 
+    def test_can_create_task_with_big_images(self):
+        # Checks for regressions about the issue
+        # https://github.com/opencv/cvat/issues/6878
+        # In the case of big files (>2.5 MB by default),
+        # uploaded files could be write-appended twice,
+        # leading to bigger raw file sizes than expected.
+
+        task_spec = {
+            "name": f"test {self._USERNAME} to create a task with big images",
+            "labels": [
+                {
+                    "name": "car",
+                }
+            ],
+        }
+
+        # We need a big file to reproduce the problem
+        image_file = generate_image_file("big_image.bmp", size=(4000, 4000), color=(100, 200, 30))
+        image_bytes = image_file.getvalue()
+        file_size = len(image_bytes)
+        assert 10 * 2**20 < file_size
+
+        task_data = {
+            "client_files": [image_file],
+            "image_quality": 70,
+            "use_cache": False,
+            "use_zip_chunks": True,
+        }
+
+        task_id, _ = create_task(self._USERNAME, task_spec, task_data)
+
+        # check that the original chunk image have the original size
+        # this is less accurate than checking the uploaded file directly, but faster
```

Comment:
Could you clarify why this is less accurate?

Reply:
Because we're not checking the original problem exactly, as it was with raw uploaded files. The chunks may get processed data, which can be correct for some foreign reasons, which can shadow the initial problem.